bgpd/bgp_table.c

```diff
@@ -69,6 +69,8 @@ static struct route_node *bgp_node_create(route_table_delegate_t *delegate,
 	struct bgp_node *node;
 	node = XCALLOC(MTYPE_BGP_NODE, sizeof(struct bgp_node));
 
+	node->soft_reconfig_table = list_new();
```

Comment:
optimisation: do not do list_new() while not used

Reply:
OK see https://github.com/FRRouting/frr/pull/7457/commits/d620906e4065e37dc7244c1dc20057f798f6765f